addons/mail/controllers/thread.py

```diff
@@ -205,8 +206,11 @@ def mail_message_post(self, thread_model, thread_id, post_data, context=None, **
         return store.add(message).get_result()
 
     @http.route("/mail/message/update_content", methods=["POST"], type="jsonrpc", auth="public")
+    @notify_bus_on_completion
     @add_guest_to_context
-    def mail_message_update_content(self, message_id, body, attachment_ids, attachment_tokens=None, partner_ids=None, **kwargs):
+    def mail_message_update_content(self, message_id, body, attachment_ids, attachment_tokens=None, partner_ids=None, context=None, **kwargs):
```

Comment:
I feel like it would be better not to change the signature of the functions using the `notify_on_bus_completion` decorator: the decorator (as currently written) pops the uuid from the kwargs, so this method will never actually receive it.